Library/Homebrew/test/test_patching.rb

```diff
@@ -136,7 +138,9 @@ def test_single_patch_dsl_with_incorrect_strip_with_apply
             sha256 TESTBALL_PATCHES_SHA256
             apply APPLY_A
           end
-        end.brew(&:patch)
+        end.brew do |f, _staging|
```

Comment:
For both brevity and clarity, I would make that `end.brew { |formula, _staging| formula.patch }`. (The shorter `f` seems also acceptable, given how widely that is used in our code base.)